app/templates/settings/users/user.hbs

```diff
@@ -130,6 +130,45 @@
           </Auk::Panel::Body>
         </Auk::Panel>
       </div>
+      <div class="auk-u-m-4 auk-u-mr-2 auk-u-ml-2">
+        <Auk::Panel>
+            <Auk::Panel::Header>
+              <AuToolbar class="auk-u-maximize-width" as |Group|>
+              <Group>
+                <h4 data-test-route-settings---user-technical-info class="auk-panel__title">
+                {{t "mandatee"}}
+                </h4>
+              </Group>
+              <Group>
+                {{#if this.selectedPerson.id}}
+                  {{#if (eq this.selectedPerson.id this.model.person.id)}}
+                  <AuButton
+                      @alert={{true}}
+                      {{on "click" this.unlinkPerson}}
+                      >
+                      {{t "unlink"}}
+                  </AuButton>
+                  {{else}}
+                  <AuButton
+                      @alert={{true}}
+                      {{on "click" this.linkPerson}}
+                      >
+                      {{t "connect"}}
+                    </AuButton>
+                    {{/if}}
+                {{/if}}
+              </Group>
+              </AuToolbar>
+          </Auk::Panel::Header>
+          <Auk::Panel::Body class="au-u-padding-none">
```

Comment:
It looks a little better if you leave the default padding in.

Reply:
hmm, somewhat disagree. No padding means the borders match up. 
![image](https://user-images.githubusercontent.com/22245223/233616909-988e8ce9-ce5e-489f-aaf4-b8caa63fa402.png)


Default padding means floating lines

![image](https://user-images.githubusercontent.com/22245223/233616842-abbee18f-95fb-44ad-8edb-a1c73ab9d293.png)


But styling without a prototype/design is not easy.
This can be fine-tuned later.

Reply:
That's a different view. I'm talking about the view in the user detail. You're showing the organization detail.

No padding:
![image](https://user-images.githubusercontent.com/22411874/233618731-74618446-c9b9-4808-85cc-a0e37a4f55af.png)

With padding:
![image](https://user-images.githubusercontent.com/22411874/233618864-14f55dc3-238d-4bf2-a521-54cf331b06b0.png)